<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMaterialWidget</class>
 <widget class="QWidget" name="AddMaterialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Material* :</string>
   </property>
  </widget>
  <widget class="TreeBox" name="materialComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>40</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLength</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventory number :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inventoryNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Individual* :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Parity :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="parityComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="individualComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Completeness :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="completenessComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="commentsLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments :</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeBox</class>
   <extends>QComboBox</extends>
   <header>CustomWidgets/treebox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
